x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/components/getting_started/languages/go.ts

```diff
@@ -11,31 +11,42 @@ import { Languages, LanguageDefinition } from '@kbn/search-api-panels';
 import { docLinks } from '../../../../../../shared/doc_links';
 
 export const goDefinition: LanguageDefinition = {
-  buildSearchQuery: `searchResp, err := es.Search().
-  Index("books").
-  Q("snow").
-  Do(context.Background())
+  buildSearchQuery: ({ indexName }) => `searchResp, err := es.Search(
```

Comment:
@Anaethelion if you have a moment can you give these go example a 👀  these are for dedicated not serverless. 

I think they're good, but would like a backup set of 👀